File: apis/Hello.js
let API = require('./API')

class Hello extends API._class {
  constructor (socket, message) {
    super(socket, message)
  }
  exec (usrname) {
    this.socket.emit(this.message, `Hello, ${usrname}!`)
  }
}

module.exports = API._creator(Hello)

File: apis/hello.py
from flask_socketio import emit
from .api import API

class apiClass(API):
  def __init__(self, socket, message):
    API.__init__(self, socket, message)

  def exec(self, usrname):
    self.socket.emit(self.message, 'Hello ' + usrname + '!', namespace='/apis')